brigade.js

```diff
@@ -0,0 +1,27 @@
+const { events, Job } = require("@brigadecore/brigadier");
+const { Check } = require("@brigadecore/brigade-utils");
+
+const projectName = "buck";
+const rustImg = "rust:1.38";
+
+function build(event, project) {
+    var build = new Job(`${projectName}-build`, rustImg);
+    build.tasks = [
+        "cd /src",
+        "cargo test",
+    ];
+    return build;
+}
+
+function runSuite(e, p) {
+    var check = new Check(e, p, build(e, p));
```

Comment:
One suggested update: we can add the build url as the final argument to this `new Check(...)` creation, like so:

```suggestion
    var check = new Check(e, p, build(e, p), `https://brigadecore.github.io/kashti/builds/${e.buildID}`);
```

Then the Checks UI will nav to the build rather than the default `brigade.sh` URL when none is supplied.